src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -1251,14 +1251,31 @@ await notificationService.PublishUpdateAsync(cr.ModelResource, s => s with
         }
     }
 
-    private async Task CreateExecutableAsync(AppResource er, ILogger resourceLogger, CancellationToken cancellationToken)
+    private async Task PublishConnectionStringAvailableEvent(IResource resource, CancellationToken cancellationToken)
```

Comment:
Do you need this still?

Reply:
Yes we still have resources that depend on it (Postgres/Redis etc).